### DISCUSSION TITLE
Code samples?

### DISCUSSION THREAD

**Discussion by matt416** [sanity-io/sanity#3349] [Sanity Studio Support] [answered]
Are there any code samples available for V3 that get into custom components?  I've seen some very cool stuff done in the demo videos, but  I'm struggling to wrap my head around it as there is limited documentation and examples available

**Comment by kmelve** [comment, marked as answer]
Hi! We're working on documenting this now! We'll have examples for it real soon!

**Reply by matt416** [reply]
Thank you! I realize this is no small task, just very excited to give it a try
